src/ApolloProvider.tsx

```diff
@@ -33,8 +38,8 @@ export default class Provider extends Component<ProviderProps, any> {
     client: PropTypes.object.isRequired,
   };
 
-  public store: Store<any>;
-  public client: ApolloClient;
+  public store:Store<any>;
```

Comment:
This should have a space, I thought TSLint enforced this. @abhiaiyer91 you might want to set up TSLint in your editor.